stat_test.go

```diff
@@ -13,6 +13,51 @@ import (
 	"github.com/gonum/floats"
 )
 
+func ExampleCircularMean() {
+	x := []float64{0, 2 * math.Pi, 4 * math.Pi, 6 * math.Pi, 8 * math.Pi}
+	weights := []float64{1, 2, 3, 4, 5}
+	cmean := CircularMean(x, weights)
+
+	fmt.Printf("The circular mean is %.5f.\n", cmean)
+	// Output:
+	// The circular mean is 0.00000.
+}
+
+func TestCircularMean(t *testing.T) {
+	for i, test := range []struct {
+		x   []float64
+		wts []float64
+		ans float64
+	}{
+		{
+			x:   []float64{0, 2 * math.Pi},
+			ans: 0,
+		},
+		{
+			x:   []float64{0, 0.5 * math.Pi},
+			ans: 0.78539816339744,
```

Comment:
Usually we do have a comment that we got the numbers from scipy (see the link I posted above).